[
    {
        "repo_name": "opencv/opencv",
        "CVE_ID": "CVE-2017-14136",
        "Problem_Type": "CWE-787",
        "Description": "OpenCV (Open Source Computer Vision Library) 3.3 has an out-of-bounds write error in the function FillColorRow1 in utils.cpp when reading an image file by using cv::imread. NOTE: this vulnerability exists because of an incomplete fix for CVE-2017-12597.",
        "URL": "https://github.com/opencv/opencv/pull/9448",
        "Tag": "['Issue Tracking', 'Patch', 'Third Party Advisory']",
        "fix_commit_hash": "aacae2065744adb05e858d327198c7bbe7f452b0",
        "inducing_commit_hash_pyszz": [
            "6172803012c679ec345c2e91207874d72ec62a89"
        ],
        "inducing_commit_hash_pd": [
            "f78a3b4cc19c76bf93348e5666f82d353bae2a55",
            "6172803012c679ec345c2e91207874d72ec62a89",
            "8a4a1bb018426087d332ab2cfe859b5cdc521620",
            "127d6649a1c83397bf42799ac807af41aa507b30"
        ],
        "matched": [
            "6172803012c679ec345c2e91207874d72ec62a89"
        ]
    },
    {
        "repo_name": "opencv/opencv",
        "CVE_ID": "CVE-2017-17760",
        "Problem_Type": "CWE-119",
        "Description": "OpenCV 3.3.1 has a Buffer Overflow in the cv::PxMDecoder::readData function in grfmt_pxm.cpp, because an incorrect size value is used.",
        "URL": "https://github.com/opencv/opencv/pull/10369/commits/7bbe1a53cfc097b82b1589f7915a2120de39274c",
        "Tag": "['Issue Tracking', 'Patch', 'Third Party Advisory']",
        "fix_commit_hash": "7bbe1a53cfc097b82b1589f7915a2120de39274c",
        "inducing_commit_hash_pyszz": [
            "797030aa901f3ddc64440b90e7ec1a270d988c93"
        ],
        "inducing_commit_hash_pd": [
            "797030aa901f3ddc64440b90e7ec1a270d988c93"
        ],
        "matched": [
            "797030aa901f3ddc64440b90e7ec1a270d988c93"
        ]
    },
    {
        "repo_name": "opencv/opencv",
        "CVE_ID": "CVE-2019-15939",
        "Problem_Type": "CWE-369",
        "Description": "An issue was discovered in OpenCV 4.1.0. There is a divide-by-zero error in cv::HOGDescriptor::getDescriptorSize in modules/objdetect/src/hog.cpp.",
        "URL": "https://github.com/opencv/opencv/pull/15382",
        "Tag": "['Exploit', 'Patch', 'Third Party Advisory']",
        "fix_commit_hash": "5a497077f109d543ab86dfdf8add1c76c0e47d29",
        "inducing_commit_hash_pyszz": [
            "9a62df16507fc695a93a0e61b12e1740e1428c00"
        ],
        "inducing_commit_hash_pd": [
            "806aab164b62eb8249103882a85868a13aff2d52",
            "9a62df16507fc695a93a0e61b12e1740e1428c00",
            "127d6649a1c83397bf42799ac807af41aa507b30"
        ],
        "matched": [
            "9a62df16507fc695a93a0e61b12e1740e1428c00"
        ]
    }
]